test/index.js

```diff
@@ -1,24 +1,84 @@
+'use strict';
+
 const assert = require('assert');
 const nock = require('nock');
-const {verify} = require('../index');
 
-// mock the request
-nock('https://hcaptcha.com')
-  .post('/siteverify', 'secret=secret&response=token')
-  .reply(200, '{"success":true}')
+nock.disableNetConnect();
+
+const {verify} = require('../index');
 
 describe('hCaptcha', function () {
-  describe('verify', function () {
-    const secret = 'secret';
-    const token = 'token';
-
-    it('resolves on success', function (done) {
-      verify(secret, token)
-        .then((data) => {
-          assert.deepStrictEqual(data, {success: true});
-          done();
-        })
-        .catch(done);
+    describe('verify', function () {
```

Comment:
Looks like the code uses 2 spaces for indentation instead of the 4 (even configured in the EditorConfig). Please update the indentation back to 2 spaces, thanks.

Reply:
not sure why intellij is ignoring it, i'll take a look.

Reply:
oh right, the test was written with 2 spaces while the editorconfig is configured to 4.. is it ok if i update editorconfig to 2?

Reply:
Yeah, your configuration was working as expected :) but since the project already used 2 spaces let's continue doing that and let's update the editorconfig to use 2 spaces.

Reply:
done, should be good to go.